dockers/docker-platform-monitor/supervisord.conf

```diff
@@ -45,3 +45,12 @@ autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 startsecs=0
+
+[program:xcvrd]
+command=/usr/bin/xcvrd
+priority=6
```

Comment:
I try to align the priority with the order in which we'd like the applications to start. Because supervisor doesn't handle sequential starting properly, we start our programs in the start.sh script. There, you have xcvrd starting after rsyslogd. Please adjust one of the files so that they are in agreement.

Reply:
move xcvrd to be started after ledd in start.sh